index.md

```diff
@@ -1,48 +1,58 @@
+todo: make this
+
+
 [![DOI](https://zenodo.org/badge/6034062.svg)](https://zenodo.org/badge/latestdoi/6034062)
-## The BrainGrid Project
 
-The original idea behind the BrainGrid Project is to develop a toolkit/software architecture to ease creating high-performance neural network simulators. It is now composed of two parts BrainGrid (the simulator) and Workbench (a simulation and data/software provenance tool). So, we now say that BrainGrid+Workbench focuses on enabling **high-performance/high-quality** neural simulation. Its key features:
+## The Graphitti Project
+
+The original idea behind the Graphitti Project is to develop a toolkit/software architecture to ease creating high-performance network simulators. It is now composed of two parts Graphitti (the simulator) and Workbench (a simulation and data/software provenance tool). So, we now say that Graphitti+Workbench focuses on enabling **high-performance/high-quality** neural simulation.
+
+### [Graphitti](https://uwb-biocomputing.github.io/Graphitti/)
+
+A project to facilitate construction of high-performance neural simulations.
```

Comment:
Change to "A project to facilitate construction of high-performance simulations of network-structured systems."